issue_02/script.js

```diff
@@ -0,0 +1,11 @@
+const ul = document.getElementById('js-ul');
+const li = document.createElement('li');
+const a = document.createElement('a');
+const img = document.createElement('img');
+
+img.src = 'bookmark.png';
+img.alt = 'ブックマーク';
+a.href = '1.html';
+a.textContent = 'これです';
```

Comment:
I don't think there's anything wrong with the code.
I'll share what I learned when I did this assignment before.

When setting attribute values, as Yoshikawa-san writes in these lines, there are two ways to write them: by specifying properties or by using setAttribute.
He says that setAttribute is used when you want to give a custom attribute that the DOM doesn't have.

Reply:
**Thank you for your comment and review!**

I've never heard of `setAttribute` before.
According to the [documentation](https://developer.mozilla.org/ja/docs/Web/API/Element/setAttribute), it seems to be a good idea to use it when creating a new attribute.
I'll try to use it in the future when I get the chance!